src/Core/Search.php

```diff
@@ -52,7 +52,7 @@ public static function getContactsFromProbe($user)
 			}
 
 			$contactDetails = Contact::getDetailsByURL(defaults($user_data, 'url', ''), local_user());
-			$itemUrl = (($contactDetails["addr"] != "") ? $contactDetails["addr"] : defaults($user_data, 'url', ''));
+			$itemUrl        = (($contactDetails["addr"] != "") ? $contactDetails["addr"] : defaults($user_data, 'url', ''));
```

Comment:
This can be rewritten to 
```php
$itemUrl = defaults($contactDetails, 'addr', defaults($user_data, 'url', ''));
```
or
```php
$itemUrl = $contactDetails["addr"] ? : defaults($user_data, 'url', '');
```